editor/editor_node.cpp

```diff
@@ -508,6 +508,9 @@ void EditorNode::_update_from_settings() {
 
 	float lod_threshold = GLOBAL_GET("rendering/mesh_lod/lod_change/threshold_pixels");
 	scene_root->set_lod_threshold(lod_threshold);
+
+	RS::get_singleton()->decals_set_filter(RS::DecalFilter(int(GLOBAL_GET("rendering/textures/decals/filter"))));
```

Comment:
Shouldn't we include decal filter and projector light filter here?

Reply:
Nevermind. Meant to ask about the quality settings. But now I can see the quality settings is based on what sampler is used.